10up/retro-winamp-block
# Added the ability to change player skin

### Description of the Change
Added the ability to change the player skin from the sidebar settings panel. Now users will be able to select different skins from the predefined list of skins (4 at the moment), for designs please see #1 

Additionally, if users want to use any custom skin they can do so by toggling the switch on and adding any custom skin URL in the input field. Also added the thumbnails for the predefined skins for better user experience.

Closes #5 

### How to test the Change
1. Create a new post/page
2. Select the "Winamp Block" block
3. In the sidebar settings panel(under Winamp Player Skin), you'll see the new predefined skins with thumbnails and a toggle switch to use custom skin

Here's a screenshot:

<img width="279" alt="predefined-skins" src="https://user-images.githubusercontent.com/45451844/187238096-9952a89b-ec39-4759-88d0-2b81451dc7d3.png">

When using custom skin input:
<img width="280" alt="custom-skin-option" src="https://user-images.githubusercontent.com/45451844/187238161-13cbe665-9e17-400e-8813-d1846c146d8c.png">


### Changelog Entry
> Added - Added the ability to change player skin from predefined list of skins with an option for custom skin input


### Credits
Props @jeffpaul @dkotter @zamanq 


### Checklist:
